src/applications/static-pages/sidenav-behavior.js

```diff
@@ -0,0 +1,45 @@
+/*
+First accordion item open by default
+*/
+
+export function openFirstLeftNavItem(sidenav) {
```

Comment:
Is it possible to do this at build time, instead? We have the url, it seems like we could avoid doing this client side and having that flash where the menu is closed, then open.

Reply:
I'm down. Could you point me towards an example of integrating something like this into the build?

Reply:
I don't think we have anything exactly like this. The old sidebar template kind of does it, but it's not exactly the same. I think you'd need to update the sidebar templates to set `aria-expanded` and `aria-hidden` on the accordion buttons and sections depending on the loop index and `entityUrl.path`.

Reply:
Was able to implement this in the sidebar templates.